### ISSUE TITLE
[BUG] JRM doesn't save settings or state until app close

### ISSUE THREAD

**Issue by 1dolla** [optyfr-org/JRomManager#287]
There's no "Save" or "OK" or "Apply" button, yet if I change any setting and perform a scan, nothing is saved (except my profile still exists), and everything is reset if the app crashes and is relaunched, so I have to start over from scratch. If I change settings, then quit and relaunch, it sticks.

**Comment by optyfr** [comment]
true, but it shouldn't crash either, so why does it crash? Not the fact that I don't want to add a "save setting" button somewhere, but it's like trying to circumvent another problem...

**Comment by 1dolla** [comment]
Well, it could crash for a number of reasons. There are different runtimes out there, there are power outages, there are overclocked computers, space rays, coffee mugs, cats on keyboards, angry wives, etc. Also, there are bugs on your end that you don't know about.

I've never come across a program that saves on exit. Why do I need to exit the program and restart it for my settings to save? Programs either seem to do Steve Jobs and save whenever you change something (which I don't like, even though I'm currently a macOS user), or Redmond or Linux and save when you click save (which I like, because it means I decide, and I'm old and schooled by Windows and Linux).

:)
